docs/source/markdown/podman-pod-clone.1.md.in

```diff
@@ -67,6 +67,10 @@ Set a custom name for the cloned pod. The default if not specified is of the syn
 
 @@option pid.pod
 
+@@option restart
+
+The restart policy is set at the pod level so all containers in the pod have the same restart policy configured.
```

Comment:
The description isn't accurate anymore since containers can still use a different policy if the user wishes to.  How about:
`Default restart policy for all container in the pod.`?

Reply:
Fixed